streaming.md

```diff
@@ -23,8 +23,8 @@ Livestreaming is the equivalent of television broadcasting on the internet. Ther
 families of servers for livestreaming: RTP/RTSP and HTTP. Matroska is not meant to be
 used over RTP. RTP already has timing and channel mechanisms that would be wasted if doubled
 in Matroska. Additionally, having the same information at the RTP and Matroska level would
-be a source of confusion if they do not match. Livestreaming of Matroska over HTTP
-(or any other plain protocol based on TCP) is possible.
+be a source of confusion if they do not match.
+Livestreaming of Matroska over file-like protocols like HTTP, QUIC, FTP, etc is possible.
```

Comment:
Personally I wouldn't even mention FTP here. FTP is an anachronism that isn't even widely used anymore (and really shouldn't from a security point of view). Just say "…over file-like protocols such as HTTP or QUIC is possible" and leave it at that.

Reply:
`etc` > `etc.`

I agree that FTP is an anachronism, but still I see it used quite often. This is just a remark and **not** an objection for deleting it here, of course.

Reply:
I added FTP as a replacement for the "TCP based" protocol. But then QUIC is over UDP. I'll just drop it.